lib/flex/FlexController.js

```diff
@@ -150,7 +151,10 @@ class FlexController extends BaseController {
         this.projectSetup.save(next);
       }
     ], (err) => {
-      this.cliManager.processCommandResult(err, null, done);
+      const cmdResult = new CommandResult()
```

Comment:
We are using es6 classes and I really like the syntactic sugar they offer :blush: Furthermore, I believe one would expect to 'instantiate' them using `new className()` rather than `className.new()`.